lib/tasks/gettext.rake

```diff
@@ -90,4 +90,25 @@ namespace :gettext do
       end
     end
   end
+
+  task :extract_locale_names do
+    require 'yaml/store'
+    require_relative '../vmdb/fast_gettext_helper.rb'
```

Comment:
If rails is loaded, you can do `require Rails.root.join("vmdb/fast_gettext_helper")`